src/main.c

```diff
@@ -2763,6 +2793,8 @@ int craft_main(int argc, char **argv) {
         me->name[0] = '\0';
         me->buffer = 0;
         g->player_count = 1;
+        //bool allow_auto_camera_keyPress = true;
+        //bool camera_auto_enabled = false;
```

Comment:
Same complaint as before. It is not good practice to have commented out code, because I am not sure if you did this for a particular reason that is for program functionality or it is just a development byproduct. Please remove this commented out code if it is not of any use.

Reply:
unwanted commented code is removed.